Monika After Story/game/zz_pianokeys.rpy

```diff
@@ -807,6 +832,110 @@ init 1000 python in mas_piano_keys:
     # _pnm_<song name inital>_v#l#
     # v#l# -> verse #, line #
 
+
+###  HAPPY BIRTHDAY ###########################################################
+## full combo
+## w w e w t r
+## w w e w y t
+## w w o u t r e
+## i i u t y t
+
+    # happy birthday, piano note setup
+    _pnm_hb_v1l1 = PianoNoteMatch(
+        renpy.text.text.Text(
+            "Happy Birthday to you",
+            style="monika_credits_text"
+        ),
+        [
+            G4,
+            G4,
+            A4,
+            G4,
+            C5,
+            B4,
+        ],
+        postnotes=None,
+        express="1k",
+        postexpress="1a",
+        verse=0
+    )
+
+    _pnm_hb_v1l2 = PianoNoteMatch(
+        renpy.text.text.Text(
+            "Happy Birthday to you~",
```

Comment:
Tildes look bad in Monika's credit font. maybe use an ! instead? 

tbh, just removing the tilde and having a different expression might be more intuitive.